ios/browser/api/ads/brave_ads.mm

```diff
@@ -475,13 +475,15 @@ - (void)applicationDidBecomeActive {
     return;
   }
   adsClientNotifier->NotifyBrowserDidEnterForeground();
+  adsClientNotifier->NotifyBrowserDidBecomeActive();
 }
 
 - (void)applicationDidBackground {
   if (![self isAdsServiceRunning]) {
     return;
   }
   adsClientNotifier->NotifyBrowserDidEnterBackground();
+  adsClientNotifier->NotifyBrowserDidResignActive();
```

Comment:
Does the order of this matter? I.e. in the did become active you do `enter foreground` → `become active`.  On this side its `entered background` → `resign active` but that's technically backwards

Reply:
It should not matter, however I have made the order the same, and improved a couple other parts of this PR. Thanks